qa/rpc-tests/bumpfee.py

```diff
@@ -213,6 +213,10 @@ def test_unconfirmed_not_spendable(rbf_node, rbf_node_address):
     assert bumpid not in rbf_node.getrawmempool()
     assert rbfid in rbf_node.getrawmempool()
 
+    # call abandontransaction to make sure the wallet does not add the bump
+    # transaction back into the mempool.
+    rbf_node.abandontransaction(bumpid)
```

Comment:
This should take place after `submit_block_with_tx` and before `invalidateblock`
That is the only time it is guaranteed that bumpid is not in the mempool.